packages/stable/src/api/documents/documentsApi.ts

```diff
@@ -0,0 +1,49 @@
+// Copyright 2022 Cognite AS
+
+import { BaseResourceAPI, CDFHttpClient, MetadataMap } from '@cognite/sdk-core';
+
+import {
+  Document,
+  DocumentSearchResponse,
+  DocumentSearchRequest,
+} from '../../types';
+
+export class DocumentsAPI extends BaseResourceAPI<Document> {
+  constructor(...args: [string, CDFHttpClient, MetadataMap]) {
```

Comment:
I think the constructor can be removed as it doesn't do anything beyond the default behavior.

Reply:
Done